docs/source/markdown/podman-events.1.md

```diff
@@ -142,11 +142,17 @@ $ sudo podman events --since 5m
 
 Show Podman events in JSON Lines format
 ```
-events --format json
+$ podman events --format json
 {"ID":"683b0909d556a9c02fa8cd2b61c3531a965db42158627622d1a67b391964d519","Image":"localhost/myshdemo:latest","Name":"agitated_diffie","Status":"cleanup","Time":"2019-04-27T22:47:00.849932843-04:00","Type":"container"}
 {"ID":"a0f8ab051bfd43f9c5141a8a2502139707e4b38d98ac0872e57c5315381e88ad","Image":"docker.io/library/alpine:latest","Name":"friendly_tereshkova","Status":"unmount","Time":"2019-04-28T13:43:38.063017276-04:00","Type":"container"}
 ```
 
+```
+$ podman events --stream=false --format={{json.}}
```

Comment:
I don't know if I want to advertise this in the manpages - the preferred way should be `--format json`

Reply:
Removed from the manpage.